### PR TITLE
feat(workflow): support limit count action

### PR DESCRIPTION
**Pull request by tzssangglass**
### Description

Wait for #7760 merged

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: apisix/plugins/workflow.lua [hunk 1]
```diff
@@ -82,6 +86,10 @@ local support_action = {
         handler  = exit,
         schema   = return_schema,
     },
+    ["limit-count"] = {
+        handler  = rate_limit,
+        schema   = limit_count.schema,
```
Comment (spacewander) [inline apisix/plugins/workflow.lua:91]: We can't use group in the schema. As using group in the workflow will affect other limit-count configuration.

Comment (tzssangglass) [reply apisix/plugins/workflow.lua:91]: fixed

File: apisix/plugins/workflow.lua [hunk 2]
```diff
@@ -117,6 +117,10 @@ function _M.check_schema(conf)
                 return false, "failed to validate the '" .. action[1] .. "' action: " .. err
             end
 
+            if action[1] == "limit-count" and action[2]["group"] then
```
Comment (spacewander) [inline apisix/plugins/workflow.lua:120]: Better to put it in the schema checker of limit-count.

**Review by spacewander — DISMISSED** [review]

**Comment by spacewander** [comment]
Please make the CI pass, thanks!

**Comment by spacewander** [comment]
BTW, it would be better also to expose the schema so 3rd party tools like dashboard can reuse the validation.

File: apisix/plugins/limit-count/init.lua [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+
+function _M.check_schema(conf)
+    local ok, err = core.schema.check(schema, conf)
+    if not ok then
+        return false, err
+    end
+
+    if conf.group then
+        -- means that call by some plugin not support
+        if conf._vid then
```
Comment (soulbird) [inline apisix/plugins/limit-count/init.lua:152]: `vid` is injected in `workflow` plugin, it will be better validate in a plugin.

Comment (tzssangglass) [reply apisix/plugins/limit-count/init.lua:152]: ref: https://github.com/apache/apisix/pull/7771#discussion_r955692338

**Review by soulbird — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**